components/Slideshow/Upload.js

```diff
@@ -0,0 +1,86 @@
+import { Component } from "react"
+import React from "react"
+import Dropzone from "react-dropzone"
+import axios from "axios"
+
+class Upload extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      SLIDE_LIST: []
+    }
+  }
+
+  handleOnDrop = (acceptedfiles, rejectedFiles) => {
+    //eslint-disable-next-line
+    console.log("accepted" + acceptedfiles)
+    //eslint-disable-next-line
+    console.log("rejected" + rejectedFiles)
+  }
+
+  handleOnDropAccepted = acceptedFiles => {
+    /* eslint-disable */
```

Comment:
You can enable the linter for this block, just disable it for the `console.log` line at the end